### PR TITLE
New workflow callbacks

### PR DESCRIPTION
**Pull request by eder7**
* onBeforeSetup
* onAfterSetup
* onAfterTearDown

### CODE REVIEW

File: sweetest/src/main/java/com/mysugr/sweetest/framework/cucumber/HookOrder.kt [hunk 1]
```diff
@@ -3,7 +3,12 @@ package com.mysugr.sweetest.framework.cucumber
 object HookOrder {
 
     const val INITIALIZE_FRAMEWORK = 0
-    const val INITIALIZE_STEPS = 10000
-    const val INITIALIZE_DEPENDENCIES = 20000
-    const val SETUP = 30000
-}
\ No newline at end of file
+    const val INITIALIZE_STEPS = 10_000
```
Comment (ChristopherKoellner) [inline sweetest/src/main/java/com/mysugr/sweetest/framework/cucumber/HookOrder.kt:6]: Just out of interest, where do this numbers come from?

**Comment by eder7** [comment]
Checked on both mySugr repositories, added usage of some of the new callbacks, runs smoothly ✅

**Review by ChristopherKoellner — APPROVED** [review]
LGTM

File: sweetest/src/main/java/com/mysugr/sweetest/framework/cucumber/HookOrder.kt (hunk 1 above)
Comment (eder7) [reply sweetest/src/main/java/com/mysugr/sweetest/framework/cucumber/HookOrder.kt:6]: there are arbitraty order numbers, but they are important for ordering the Cucumber hooks (see [here](https://cucumber.io/docs/cucumber/api/#hooks)).

**Pull request merged**